tests/pyccel/test_pyccel.py

```diff
@@ -27,17 +28,36 @@ def get_abs_path(relative_path):
     return os.path.join(base_dir, relative_path)
 
 #------------------------------------------------------------------------------
-def get_exe(filename):
+def get_exe(filename, language=None, prog=False):
     exefile = os.path.splitext(filename)[0]
-    if sys.platform == "win32":
-        exefile = exefile + ".exe"
-    if not os.path.isfile(exefile):
-        dirname = os.path.dirname(filename)
-        basename = "prog_"+os.path.basename(filename)
-        exefile = os.path.join(dirname, os.path.splitext(basename)[0])
+
+    if language == 'python':
+        result = ''
+        if sys.platform == "win32":
+            result = exefile.split('\\')
```

Comment:
You can use `os.path.basename(filename)` to do this in a system independent way (see original code)